src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -817,6 +817,13 @@ function getTransformHandles(transformBounds, margin) {
 }
 
 function getTransformSelectionBounds(glyph, selection, getBackgroundImageBoundsFunc) {
+  const { point: pointIndices } = parseSelection(selection);
```

Comment:
Please write this so that parseSelection is only called when selection.size equals 1. Something like this maybe:

```js
   if (selection.size == 1 && parseSelection(selection).point?.length == 1) {
```

Reply:
Good point. I changed it as recommended and also tested it. Works well. Done.